selfdrive/car/gm/interface.py

```diff
@@ -44,6 +44,26 @@ def get_steer_feedforward_function(self):
     else:
       return CarInterfaceBase.get_steer_feedforward_default
 
+  @staticmethod
+  def torque_from_lateral_accel_bolteuv(lateral_accel_value, torque_params, lateral_accel_error, lateral_accel_deadzone, friction_compensation, v_ego, g_lat_accel):
+    ANGLE_COEF = 0.16179233
+    ANGLE_COEF2 = 0.20691964
+    SPEED_OFFSET = -7.94958973
+    SIGMOID_COEF_RIGHT = 0.34906506
+    SIGMOID_COEF_LEFT = 0.20000000
+    SPEED_COEF = 0.38748798
+
+    x = ANGLE_COEF * lateral_accel_value * (40.23 / (max(0.2,v_ego + SPEED_OFFSET))**SPEED_COEF)
+    sigmoid = erf(x)
+    out = ((SIGMOID_COEF_RIGHT if lateral_accel_value < 0. else SIGMOID_COEF_LEFT) * sigmoid) + ANGLE_COEF2 * lateral_accel_value
+    return out + g_lat_accel
```

Comment:
Can you simplify this and find a common tune across all GM? They're likely all very similar

Reply:
I noticed that Acadia was added, let me know if you need me to test it.